### PR TITLE
Updated controls-theory Readme.md

### PR DESCRIPTION
**Pull request by aditya-shirwatkar**
* Added Course info's and reviews for Brian Douglas series, Control Bootcamp, and Under Actuated Robotics
* Added Tools, Libraries and Frameworks section
  * Added, "Some interesting repositories to check out sub-section"

### CODE REVIEW

File: control-theory/README.md [hunk 1]
```diff
@@ -2,15 +2,30 @@
 
 ## Resources
 ### Classical Control
-* [Classical Control Theory](https://www.youtube.com/watch?v=oBc_BHxw78s&list=PLUMWjy5jgHK1NC52DXXrriwihVrYZKqjk) by Brian Douglas.
-* [Control Bootcamp](https://www.youtube.com/playlist?list=PLMrJAkhIeNNR20Mz-VpzgfQs5zrYi085m) by Steve Brunton.
-* [Control of Mobile Robots](https://www.coursera.org/learn/mobile-robot/home/welcome) by Dr. Magnus Egerstedt.
+* [Classical Control Theory](https://www.youtube.com/watch?v=oBc_BHxw78s&list=PLUMWjy5jgHK1NC52DXXrriwihVrYZKqjk) by Brian Douglas.                  
+  **_Why this lecture series? :_** 
+This collection of videos is intended to supplement a first-year controls class, not replace it. My goal is to take specific concepts in controls and expand on them to provide an intuitive understanding which will ultimately make you a better controls engineer.
```
Comment (akshaykulkarni07) [inline control-theory/README.md]: "My goal is to ...." Change to 3rd person (looks like you've copied from his course description or something).

File: control-theory/README.md [hunk 2]
```diff
@@ -2,15 +2,30 @@
...
+ 
+ 
 #### Graduate level: 
-* [Underactuated Robotics](http://underactuated.csail.mit.edu/Spring2020/) by Dr. Russ Tedrake.
+* [Underactuated Robotics](http://underactuated.csail.mit.edu/Spring2020/) by Dr. Russ Tedrake.                                                      
+**_Course info:_** Take a few examples like trajectories of a quadrotor being generated to play a game of pong, Or maybe a humanoid performing a backflip, Or for a simple case make an Acrobot stay upright. In all these cases we can observe that the control algorithms are exploiting the dynamics to perform a task no matter how bizarre it is. Well, This course will help you exploit those dynamics of any given system to perform tasks that may seem impossible from the classical control theory point of view. The course covers techniques like Dynamic Programming, LQR, Lyapunov Analysis, Trajectory Optimization, Model Predictive Control, Motion Planning as Search, Pixels to Torques, Robust and Stochastic Control to State Estimation, Reinforcement Learning and few other techniques
```
Comment (akshaykulkarni07) [inline control-theory/README.md]: "Well, This course will......" T should be lowercase.

File: control-theory/README.md [hunk 3]
```diff
@@ -25,7 +40,27 @@
...
+
+## Libraries, Tools and Frameworks
+
+* [ROS](https://www.ros.org/)
+* [Matlab & Simulink (For Windows)](https://in.mathworks.com/)
+* [CasADi](https://web.casadi.org/)
+* [Python Control Systems package](https://python-control.readthedocs.io/en/0.8.3/)
+* [C++ control toolbox](https://github.com/ethz-adrl/control-toolbox)
+* [Drake](https://drake.mit.edu/)
+* [OpenAi Gym](http://gym.openai.com/) (Some environments will need to be modified to implement control algorithms)
```
Comment (akshaykulkarni07) [inline control-theory/README.md]: OpenAI*

File: control-theory/README.md (hunk 1 above)
Comment (aditya-shirwatkar) [reply control-theory/README.md]: yeah, the old description was no so good so I decided to take his explanation.

Comment (aditya-shirwatkar) [reply control-theory/README.md]: I'll correct it

File: control-theory/README.md (hunk 2 above)
Comment (aditya-shirwatkar) [reply control-theory/README.md]: Ok cool

File: control-theory/README.md (hunk 3 above)
Comment (aditya-shirwatkar) [reply control-theory/README.md]: Corrected

**Comment by take2rohit** [comment]
@aditya-shirwatkar whats the status

**Comment by aditya-shirwatkar** [comment]
@akshaykvnit suggested some changes, which were done

**Comment by take2rohit** [comment]
If discussed with Satyam do let me know. I will merge it

**Comment by take2rohit** [comment]
Conflicts have been resolved now you can carry on with the changes!

**Comment by take2rohit** [comment]
Merging this pull request due to inactivity!
Feel free to re-open this PR in case of changes

**Pull request merged**